examples/siro_sandbox/sandbox_app.py

```diff
@@ -204,6 +216,20 @@ def local_end_episode(do_reset=False):
 
         self._reset_environment()
 
+    def _check_init_server(self, line_render):
+        self._remote_gui_input = None
+        self._interprocess_record = None
+        if do_network_server:
+            # How many frames we can simulate "ahead" of what keyframes have been sent.
+            # A larger value increases lag on the client, while ensuring a more reliable
+            # simulation rate in the presence of unreliable network comms.
+            max_steps_ahead = 3
+            self._interprocess_record = InterprocessRecord(max_steps_ahead)
```

Comment:
@0mdc you should review this server code including everything in `server/`. I'm not very experienced writing networking code so keep an eye out for bugs!